data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/InjectionRangeActionAdderImplTest.java

```diff
@@ -125,6 +125,37 @@ void testAddWithoutGroupId() {
         assertEquals(2, crac.getNetworkElements().size());
     }
 
+    @Test
+    void testAddDefaultKey() {
+        InjectionRangeAction injectionRangeAction = crac.newInjectionRangeAction()
+            .withId("id1")
+            .withOperator("BE")
+            .withGroupId("groupId1")
+            .withNetworkElement(injectionId1)
+            .withNetworkElement(injectionId2, injectionName2)
```

Comment:
I wonder if we want this to add both elements with key = 1, or only the last added element (i.e. injectionId2), with key = 1. what do you think @phiedw ?

Reply:
@wangjer can you please update the branch following these comments?